addons/account/views/account_portal_templates.xml

```diff
@@ -66,8 +66,11 @@
                                 <em t-else="">Draft Invoice</em>
                             </a>
                         </td>
-                        <td><span t-field="invoice.invoice_date"/></td>
-                        <td class='d-none d-md-table-cell'><span t-field="invoice.invoice_date_due"/></td>
+                            <td><span t-field="invoice.invoice_date"/></td>
```

Comment:
span not needed here can be a t balise directly, don't remember if we can put a t-field directly in the td but i think not

Reply:
This was the original way it was done. My diff only had some additional indentation that were probably added by auto-formatter
I think changing the <span> to <t> would be irrelevant diff. I will remove the extra unneeded indentation